### PR TITLE
PIE-1675 Analytics rest api suite#show docs 

### PR DESCRIPTION
**Pull request by amybiyuliu**
### Description
Add information about analytics Suite#show API endpoint

**Linear:** https://linear.app/buildkite/issue/PIE-1675/rest-api-endpoints-documentation-suiteshow
Screenshot 2023-06-01 at 2 39 45 pm

The endpoint is currently behind a feature flag. I'm not sure if I can also hide the link to this doc page behind a feature flag, or hold off releasing this PR until the endpoint is publicly released.

### CODE REVIEW

**Comment by mbelton-buildkite** [comment]
I see this is in draft, so I haven't at had look at the content, but for:

> I'm not sure if I can also hide the link to this doc page behind a feature flag, or hold off releasing this PR until the endpoint is publicly released.

We don't have feature flagging in the docs so it's best to release the docs as part of turning on the feature. 😊

**Review by gchan — APPROVED** [review]
LGTM. Perhaps we could add a sentence about what the Suites API does, similar to: https://buildkite.com/docs/apis/rest-api/analytics/flaky-tests.

I think we should hold off the merge/deploy until the feature is generally available. Perhaps talk to the team about the timing of releasing the API docs?

File: pages/apis/rest_api/analytics/suites.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Suites API
```
Comment (mbelton-buildkite) [inline pages/apis/rest_api/analytics/suites.md:1]: Style nitpick: Leave a blank line after headings.

```suggestion
# Suites API

```

**Review by mbelton-buildkite — APPROVED** [review]
One minor style nitpick, but otherwise, looks good to me 😊

**Comment by amybiyuliu** [comment]
I updated the base branch to `pie-1713-release-ta-rest-api-documentation` as we agreed to leave one long lived branch open for all new docs. The decision is been made here: https://3.basecamp.com/3453178/buckets/32646751/messages/6225392969#__recording_6229855011

**Pull request merged**